pybliometrics/sciencedirect/tests/test_ObjectRetrieval.py

```diff
@@ -0,0 +1,43 @@
+"""Test class ObjectRetrieval"""
+import xml.etree.ElementTree as ET
```

Comment:
newline after the docstring. Docstring with dot. Simply write "Tests for ObjectRetrieval() class."